### ISSUE TITLE
Rotate and Scrub Credentials

### ISSUE THREAD

**Issue by jcsalterego** [anantdgoel/HackPrincetonF16#1]
I'm unsure if there are any paid APIs tied to this project, but it would be wise to rotate and scrub credentials from this repository, and load them from a git ignored file.

**Comment by anantdgoel** [comment]
Yes, we realised this. We are going to clean it all up by tonight

**Comment by samuelhkahn** [comment]
Hey your credentials are still up, I definitely suggest removing them!

**Comment by lewisje** [comment]
still up
